api/src/routes/cards.route.ts

```diff
@@ -15,11 +16,16 @@ export class CardsRoute implements Routes {
   }
 
   private initializeRoutes() {
-    this.router.get(`${this.path}`, this.card.getCards);
-
     // Route for post request for creating a new card
     this.router.post(`${this.path}`, ValidationMiddleware(CreateCardDto), this.card.createCard);
 
+
+    // Route for post request for creating a new price history entry
+    this.router.post(`${this.path}/price-history`, ValidationMiddleware(CreatePriceHistoryDto), this.card.createPriceHistory)
```

Comment:
Price history path should be a sub path of the pokemon card, like `$this.path/:id/price-history`, that way you can infer the pokenmon card ID and won't have to have it in the payload.